### ISSUE TITLE
Possible to add support for 'Comment' field?

### ISSUE THREAD

**Issue by cwegener** [dlwyatt/PolicyFileEditor#18]
I have just started using the PolicyFileEditor module for my DSC work and it
does exactly what I need.

One feature that I would like to see is the ability to write into the 'Comment'
area of each Administrative Setting. That way, I can leave some notes in a
`gpresult /H` output file if anybody ever were to look at the Windows machine
that was configured via DSC + PolicyFileEditor.

Is this possible to implement at all?

If so, how complex would this be to implement?

**Comment by dlwyatt** [comment]
Hmm, don't think those fields existed back when I wrote this.  Not sure if those comments live in the same registry.pol file where the actual data is, or not.

Looks like Microsoft retired the technet gallery though, which is where I had published the C# code behind the dll that's in this project.  I'll have to look around and see if I can find it again, before that sort of update could be made.

**Comment by dlwyatt** [comment]
Yup, that code may be lost forever :\  It's something I did back before I even knew what Git was, and since it was already publicly available on the MS script gallery, I never thought to publish it here as well.

It's not too hard to just decompile the DLL and reproduce the functionality (either in another C# class, or just in raw PowerShell), since it's .NET code, if someone wants to do that at some point.

**Comment by OnceUponALoop** [comment]
It's a late response but I had the same thought about comments and realized they're handled out of band, ttored next to the GPO as `comment.cmtx` XML, e.g. `C:\Windows\System32\GroupPolicy\Machine\comment.cmtx`, keyed by policy id (namespace:policyname). 

I don't care enough to scratch that itch but I figured i'd pass it along :)
